<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GantryWindow</class>
 <widget class="QMainWindow" name="GantryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="ItemDisplayWindow" name="itemDisplay" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>441</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>171</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="homeButton">
       <property name="text">
        <string>Home</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadCalButton">
       <property name="text">
        <string>Load Calibration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="positionButton">
       <property name="text">
        <string>Poll Position</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scanButton">
       <property name="text">
        <string>Scan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="positionInput"/>
     </item>
     <item>
      <widget class="QPushButton" name="moveButton">
       <property name="text">
        <string>Custom Position</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>280</y>
      <width>621</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="statusDisplay">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>281</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>130</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="prevItemButton">
         <property name="text">
          <string>Previous</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextItemButton">
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="centerItemButton">
       <property name="text">
        <string>Center</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>181</width>
      <height>103</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QDoubleSpinBox" name="cal_x_spin"/>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="cal_y_spin"/>
     </item>
     <item>
      <widget class="QPushButton" name="clearObjectsButton">
       <property name="text">
        <string>Clear Objects</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>210</y>
      <width>181</width>
      <height>66</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QPushButton" name="startLiveButton">
       <property name="text">
        <string>Start Live Feed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopLiveButton">
       <property name="text">
        <string>Stop Live Feed</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QToolButton" name="increaseYButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>increaseX</string>
    </property>
    <property name="arrowType">
     <enum>Qt::UpArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="decreaseYButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="arrowType">
     <enum>Qt::DownArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="decreaseXButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="arrowType">
     <enum>Qt::LeftArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="increaseXButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="arrowType">
     <enum>Qt::RightArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="increaseZButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="arrowType">
     <enum>Qt::UpArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="decreaseZButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="arrowType">
     <enum>Qt::DownArrow</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>50</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QDial" name="dial"/>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>40</y>
      <width>141</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPlainTextEdit" name="x_pte"/>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="y_pte"/>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="z_pte"/>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="j4_pte"/>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="j5_pte"/>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="j6_pte"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>669</x>
      <y>160</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QToolButton" name="increasej5Button">
       <property name="text">
        <string>...</string>
       </property>
       <property name="arrowType">
        <enum>Qt::RightArrow</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="decreasej5Button">
       <property name="text">
        <string>...</string>
       </property>
       <property name="arrowType">
        <enum>Qt::LeftArrow</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="increasej4Button">
       <property name="text">
        <string>...</string>
       </property>
       <property name="arrowType">
        <enum>Qt::RightArrow</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QToolButton" name="decreasej6Button">
       <property name="text">
        <string>...</string>
       </property>
       <property name="arrowType">
        <enum>Qt::LeftArrow</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="increasej6Button">
       <property name="text">
        <string>...</string>
       </property>
       <property name="arrowType">
        <enum>Qt::RightArrow</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QToolButton" name="decreasej4Button">
       <property name="text">
        <string>...</string>
       </property>
       <property name="arrowType">
        <enum>Qt::LeftArrow</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="plainText">
        <string>J4</string>
       </property>
       <property name="backgroundVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPlainTextEdit" name="plainTextEdit_2">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="plainText">
        <string>J5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPlainTextEdit" name="plainTextEdit_3">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="plainText">
        <string>J6</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_4">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_5">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_6">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>0</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>CURRENT POSITION</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_7">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Milimetres</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_8">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Degrees</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>170</y>
      <width>81</width>
      <height>85</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QDial" name="dial_2"/>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber_2"/>
     </item>
    </layout>
   </widget>
   <zorder>itemDisplay</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>qvtkWidget</zorder>
   <zorder>statusDisplay</zorder>
   <zorder>verticalLayoutWidget_3</zorder>
   <zorder>verticalLayoutWidget_4</zorder>
   <zorder>verticalLayoutWidget_5</zorder>
   <zorder>increaseYButton</zorder>
   <zorder>decreaseYButton</zorder>
   <zorder>decreaseXButton</zorder>
   <zorder>increaseXButton</zorder>
   <zorder>prevItemButton</zorder>
   <zorder>increaseZButton</zorder>
   <zorder>decreaseZButton</zorder>
   <zorder>verticalLayoutWidget_6</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>plainTextEdit_4</zorder>
   <zorder>plainTextEdit_5</zorder>
   <zorder>plainTextEdit_6</zorder>
   <zorder>dial_2</zorder>
   <zorder>plainTextEdit_7</zorder>
   <zorder>plainTextEdit_8</zorder>
   <zorder>verticalLayoutWidget_7</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuELEC_490_Group_19_Gantry_Control">
    <property name="title">
     <string>ELEC 490 Group 19 : Gantry Control</string>
    </property>
   </widget>
   <addaction name="menuELEC_490_Group_19_Gantry_Control"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ItemDisplayWindow</class>
   <extends>QWidget</extends>
   <header>itemdisplaywindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
